cranelift/assembler-x64/meta/.rustfmt.toml

```diff
@@ -1,5 +1,5 @@
 # This extra configuration allows defining extra-long lines in
 # `src/instructions`.
-fn_call_width = 110
-max_width = 120
+fn_call_width = 130
+max_width = 130
```

Comment:
Unfortunately, it's getting harder and harder to keep instruction definitions on a single line. This PR includes some re-formatting due to this change.